src/inputs/dnstap/DnstapInputStream.cpp

```diff
@@ -60,6 +60,19 @@ void DnstapInputStream::_read_frame_stream_file()
                 _logger->warn("dnstap data is wrong type or has no message, skipping frame of size {}", len_data);
                 continue;
             }
+            if (_f_enabled[Filters::OnlyHosts]) {
+                if (d.message().has_query_address() && d.message().has_response_address()) {
+                    if (!_match_subnet(d.message().query_address()) && !_match_subnet(d.message().response_address())) {
+                        continue;
```

Comment:
please doc `// message had both query and response address, and neither matched, so filter`